### ISSUE TITLE
bug: (Youtube) Endless video buffering on only one account

### ISSUE THREAD

**Issue by tedecti** [microg/GmsCore#2354]
**Affected app**
Name: Revanced Extended
Package id: app.rvx.android.youtube

**Describe the bug**
When I use my main account (tedecti) videos not loading/starts buffering after 8-12 seconds, but when I use any of other accounts all working fine, I tried many fixes (clear cache, clear data, all permissions, disable power saving, reinstall all programs, install older versions of progs, clear yt history, sets spoof app versions, restart phone) but all of them cant help me

https://github.com/ReVanced/revanced-patches/assets/45327704/83d9b100-372f-4ac2-ad3e-1f9c791c515e

List of programs:
ReVanced Manager 1.9.25 (latest)
MicroG 0.3.1.4.240913 (latest)
ReVanced Extended 19.07.40 (latest)

**To Reproduce**
Cannot be reproduced because its error I saw only on this account

**System**
Android Version: 13

**microG**
microG Core version: 0.3.1.4.240913 (latest)
microG Self-Check results: "System spoof signature" and "Play Services (GmsCore) has correct signature" is not ticked

**Comment by LiamAB02** [comment]
I have this problem too.  Since two days ago, my main account plays videos for only 10 seconds and then stops loading.  But my other account works fine.  Does anyone know the solution?

**Comment by Vavun** [comment]
Ask @oSumAtrIX here https://github.com/ReVanced/GmsCore

**Comment by DinieHariz** [comment]
Me to.. same problem on my realme gt master after i reset my phone and download latest revanced manager and microG settings

**Comment by mvigerske** [comment]
Same discussion here: https://github.com/ReVanced/revanced-discussions/discussions/1037

**Comment by alyxme** [comment]
Clearing data & cache in microG,GMS app eventualy helps with buffering but you need to do this every time when buffering issue is appear.

**Comment by N2022062** [comment]
same issue recently

**Comment by phyx1u5** [comment]
> Clearing data & cache in microG,GMS app eventualy helps with buffering but you need to do this every time when buffering issue is appear.

this worked for me thanks!

**Comment by stickzman** [comment]
Having the same exact issue. Only installed a few days ago, worked great, then exactly this buffering issue started yesterday.

**Comment by phyx1u5** [comment]
> > Clearing data & cache in microG,GMS app eventualy helps with buffering but you need to do this every time when buffering issue is appear.
> 
> this worked for me as a temp fix as the buffering comes back every now and then, but thanks for the tip!

forgot to mention, Im having issues on my Xiaomi/Redmi Pad SE , using [com.google.android.gms-240913002-hw.apk](https://github.com/microg/GmsCore/releases/download/v0.3.1.240913/com.google.android.gms-240913002-hw.apk)

I actually tried installing all the different versions of microg including the revanced ones, but still the same issue.

**Comment by xamtra** [comment]
On my realme phone, I'm facing this issue, but my wife has the same model but it's perfectly working on her mobile without buffering. 
And for my surprise on my two other phones, Redmi note 10 and Redmi Note 7 also working perfectly without buffering even its working on my main account. 
The only difference is both Redmi phones are rooted and both realme phones are not rooted.

**Comment by Xshafi** [comment]
> .

Yes you're correct.

**Comment by bobbleheadhobo** [comment]
Is there a working solution to this issue yet?

**Comment by mvigerske** [comment]
Yes, THIS problem got fixed for me now with the new patches v4.8.1 and v4.8.2. But now the history is not tracked anymore... is this related? I did what they recommended: cleared, deleted and reinstalled microg/revancedmanager/revanced. Did the patch again, restarted, but no help. Just changing the YouTube account helps. The account is somehow spoiled?

**Comment by leo20481141** [comment]
I don't know if this will help but I fixed a similar issue https://github.com/microg/GmsCore/issues/2789 dointg this.

I did a factory reset and installed the new version of microG, but I made sure to set device registration profile to real this time (it's important to do it before enabling device registration for the first time because after that the id was already generated and before adding your google account) and to enable allow apps to find accounts before adding my google account.

So I don't know if this was fixed in the new version or if I set up microg wrong before but now it works.

**Comment by rahulchirag007** [comment]
I only did this settings and this infinite buffering problem is gone for me.

![Image](https://github.com/user-attachments/assets/b436afaa-4523-46f8-96c5-ca607aafe647)
![Image](https://github.com/user-attachments/assets/add80a71-f81f-45ce-a614-75c86e16d9a7)

**Comment by Turhvjbufv** [comment]
> I only did this settings and this infinite buffering problem is gone for me.
> 
> ![Image](https://github.com/user-attachments/assets/b436afaa-4523-46f8-96c5-ca607aafe647)
> ![Image](https://github.com/user-attachments/assets/add80a71-f81f-45ce-a614-75c86e16d9a7)

This is not the same project, please open an issue in it's related repo and not here
